scripts/pre-build.js

```diff
@@ -0,0 +1,15 @@
+// any scripts occurring before astro build goes here
+const { loadSpeakers, loadEvents, getPhotos } = require("./directus-dump");
+const fs = require("fs");
+
+const preBuild = async () => {
+  const speakers = await loadSpeakers();
```

Comment:
Is it worth batching it with Promise.all(Settled) here or not worth optimising at this point?